moped-etl/prefect/moped_test_create_flow.py

```diff
@@ -244,71 +268,55 @@ def drain_service(slug):
     ## Commission the Activity Log
 
     commission_activity_log_command = activity_log.create_activity_log_command(
-        slug=slug
+        slug=slug, upstream_tasks=[git_repo_checked_out]
     )
     deploy_activity_log = activity_log.create_activity_log_task(
         command=commission_activity_log_command
     )
 
-    ## Apply migrations
-
+    ## Apply Migrations, metadata and optional seed data
     graphql_endpoint = "https://" + migrations.get_graphql_engine_hostname(slug=slug)
 
     access_key = migrations.get_graphql_engine_access_key(slug=slug)
 
-    rm_clone = "rm -fr /tmp/atd-moped"
-    cleaned = migrations.remove_moped_checkout(command=rm_clone)
-
-    git_clone = f"git clone {GIT_REPOSITORY} /tmp/atd-moped"
-    cloned = migrations.clone_moped_repo(command=git_clone, upstream_tasks=[cleaned])
-
-    checkout_branch = migrations.get_git_checkout_command(slug=slug)
-    checked_out = migrations.checkout_target_branch(
-        command=checkout_branch, upstream_tasks=[cloned]
-    )
-
     metadata = "metadata"
 
     config = migrations.create_graphql_engine_config_contents(
         graphql_endpoint=graphql_endpoint,
         access_key=access_key,
         metadata=metadata,
-        checked_out_token=checked_out,
-    )
-
-    # what are these parentheses doing?
-    migrate_cmd = (
-        "(cd /tmp/atd-moped/moped-database; hasura --skip-update-check migrate apply;)"
-    )
-    migrate = migrations.migrate_db(
-        command=migrate_cmd, upstream_tasks=[config, graphql_endpoint_ready]
+        checked_out_token=git_repo_checked_out,
     )
 
-    consistency_cmd = "(cd /tmp/atd-moped/moped-database; hasura --skip-update-check metadata inconsistency status;)"
-    consistent_metadata = migrations.check_for_consistent_metadata(
-        command=consistency_cmd, upstream_tasks=[migrate]
-    )
-
-    settled_metadata = migrations.sleep_fifteen_seconds(
-        consistent_metadata=consistent_metadata
-    )
+    with case(do_migrations, True):
+        # what are these parentheses doing?
+        migrate_cmd = "(cd /tmp/atd-moped/moped-database; hasura --skip-update-check migrate apply; sleep 15;)"
```

Comment:
Even when I'm 1,000% positive that the script is not starting one `hasura <thing> apply` operation before the previous one is good and finished, if there isn't some amount of "cool off" time, the `graphql-engine` will get in an unworkable state. So, here we are with `sleep()`s in the subshell.